src/autobot/internal/autobot.h

```diff
@@ -36,25 +36,31 @@ class Autobot : public IAutobot, public async::Asyncable
     std::vector<ITestCasePtr> testCases() const override;
     ITestCasePtr testCase(const std::string& name) const override;
 
-    void runAll(const std::string& testCaseName) override;
-    void runFile(const std::string& testCaseName, int fileIndex) override;
+    void setCurrentTestCase(const std::string& name) override;
+    const ValCh<ITestCasePtr>& currentTestCase() const override;
+
+    void runAll() override;
```

Comment:
Maybe is it better to rename this method to runAllFiles?

Reply:
renamed